recipes/bioconductor-dexseq/pythonpath.patch

```diff
@@ -0,0 +1,16 @@
+--- inst/python_scripts/dexseq_count.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_count.py.new	2017-06-16 14:14:38.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
++
+ import sys, itertools, optparse, warnings
+ 
+ optParser = optparse.OptionParser( 
+--- inst/python_scripts/dexseq_prepare_annotation.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_prepare_annotation.py.new	2017-06-16 14:14:55.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
`condo-build` recognizes the `/opt/anaconda1anaconda2anaconda3` string as special and will replace it with the full prefix at install time ([relevant docs section](https://conda.io/docs/building/meta-yaml.html#making-packages-relocatable), second bullet point).

I prefer to use that instead of `/usr/bin/env python` to guarantee the correct interpreter is used.    In theory, `/usr/bin/env python` could point to a different `python`, depending on what else has been done to the `PATH` after activating the environment.